src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Mashine[] mashines = {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator(),
+        };
+        for (Mashine mashine : mashines) {
```

Comment:
The loop variable type `Mashine` is likely a typo. It should be `Machine` to match the corrected class name.